pkg/core/transaction/witness_scope.go

```diff
@@ -23,19 +25,18 @@ const (
 	CustomGroups WitnessScope = 0x20
 )
 
-// ScopesFromString converts string of comma-separated scopes to a set of scopes
-// (case doesn't matter). String can combine several scopes, e.g. be any of:
-// 'Global', 'CalledByEntry,CustomGroups' etc. In case of an empty string an
-// error will be returned.
-func ScopesFromString(s string) (WitnessScope, error) {
+// ScopesFromString converts string of separated by specified separator scopes
+// to a set of scopes (case-sensitive). String can combine several scopes, e.g.
+// be any of: 'Global', 'CalledByEntry,CustomGroups' etc. In case of an empty
+// string an error will be returned.
+func ScopesFromString(s, separator string) (WitnessScope, error) {
```

Comment:
We technically have one separator which is comma, whitespace can just be ignored, IMO.